src/input/input.jsx

```diff
@@ -41,13 +39,15 @@ const types = [
   'week',
 ]
 
+const validateRequired = [value => (value ? undefined : 'required')]
```

Comment:
You describe that we need to have a new array for every instance.
This is not the case here.

Maybe you have to create a React.Component class to handle this case.

Reply:
Well, my bad, I wasn't clear enough: we need to have a different and unique array whenever the validation function or function list is different. In that case the default for required fields will always be the same, so it can stay that way.
However, if we want to customize a required message, we will have to pass now a `validate` prop which will override the default one.
This `validate` prop should be declared outside any component (so that it is constant). And that can lead to having to declare as much `validate` arrays as Input components in our form.

Reply:
Can you add this as a comment in the Input source ?